### PR TITLE
strongswan: collapse menu items

### PR DESCRIPTION
**Pull request by mwarning**
Signed-off-by: Moritz Warning moritzwarning@web.de
Maintainer: @stintel

Description:
Strongswan occupies around 100 items in menu Netwok/VPN.
This is an attempt to cleanup the items.

### CODE REVIEW

**Comment by stintel** [comment]
With this change, strongswan doesn't appear at all, probably diue to the problems shown here:

```
tmp/.config-package.in:48110:error: recursive dependency detected!
For a resolution refer to Documentation/kbuild/kconfig-language.txt
subsection "Kconfig recursive dependency limitations"
tmp/.config-package.in:48110:   symbol PACKAGE_strongswan-full depends on PACKAGE_strongswan
For a resolution refer to Documentation/kbuild/kconfig-language.txt
subsection "Kconfig recursive dependency limitations"
tmp/.config-package.in:48006:   symbol PACKAGE_strongswan is selected by PACKAGE_strongswan-charon
For a resolution refer to Documentation/kbuild/kconfig-language.txt
subsection "Kconfig recursive dependency limitations"
tmp/.config-package.in:48032:   symbol PACKAGE_strongswan-charon is selected by PACKAGE_strongswan-charon-cmd
For a resolution refer to Documentation/kbuild/kconfig-language.txt
subsection "Kconfig recursive dependency limitations"
tmp/.config-package.in:48048:   symbol PACKAGE_strongswan-charon-cmd is selected by PACKAGE_strongswan-full
```

**Comment by mwarning** [comment]
ah, I missed that. I have to do some advanced head scratching to solve this.

File: net/strongswan/Makefile [hunk 1]
```diff
@@ -109,6 +109,7 @@ include $(INCLUDE_DIR)/nls.mk
 define Package/strongswan/Default
   SUBMENU:=VPN
   SECTION:=net
+  MENU:=1
```
Comment (cotequeiroz) [inline net/strongswan/Makefile]: This is meant to be in `Package/strongswan`, although I'm not sure it hurts to be here.

Comment (mwarning) [reply net/strongswan/Makefile]: Done, thanks!

**Comment by stintel** [comment]
Nice, thanks!

**Comment by mwarning** [comment]
Awesome :-)

**Pull request merged**